cmd/minikube/cmd/service.go

```diff
@@ -148,6 +148,10 @@ You may select another namespace by using 'minikube service {{.service}} -n <nam
 					out.String(fmt.Sprintf("%s\n", serviceURLs))
 				}
 			}
+			// check whether all pods of this service is crashed
```

Comment:
```suggestion
			// check whether there are running pods for this service
```
Pods could be pending and this could fail, so update comment to clarify this